CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
 ## [Unreleased] - ReleaseDate
+### Added
```

Comment:
Please fix the CHANGELOG.  There was a merge error, and now there are two "Added" sections.